open_alchemy/build/__init__.py

```diff
@@ -218,16 +250,105 @@ def dump(
         raise exceptions.BuildError(str(exc)) from exc
 
 
-def execute(*, spec: typing.Any, name: str, path: str) -> None:
+def build_dist(name: str, path: str, format_: PackageFormat) -> None:
+    """
+    Build a distibution package.
+
+    The formats can be combined with the bitwise operator "or" (``|``), for
+    instance, building both sdist and wheel packages can be specified like this:
+
+    .. code-block:: python
+
+        format_=PackageFormat.SDIST|PackageFormat.WHEEL
+
+    Args:
+        path: The package directory.
+        format: The package format to build.
+    """
+    pkg_dir = pathlib.Path(path) / name
```

Comment:
My preference would be to push the calculation of the output path to as late as possible and make `path` and `name` an argument to `build_sdist` and `build_wheel`. If you like, you could add a small utility function that combines them so you don't have to repeat code.

Reply:
Got it. I moved it closer to the execution of the `run()` function. I did not create a function just to join a path, it seemed a bit excessive.